doc/install/stampede.tex

```diff
@@ -13,32 +13,30 @@
 
 
 
-Setting up ASPECT on TACC Stampede is much easier than on previous systems since
+\noindent Setting up ASPECT on TACC Stampede is much easier than on previous systems since
```

Comment:
I usually think that just using the default style is the way to go. If you think about having a paragraph break but not use an indent, maybe no paragraph break is necessary to begin with. But then I'm not feeling strongly enough about it to put on a fight :-)